pages/index.tsx

```diff
@@ -35,7 +36,8 @@ export default function Home({
         <section>
           <TestManufacturerSelector manufacturers={manufacturers} />
           <h2>Original Expiration Date</h2>
-          <OriginalExpirationDate />
+          <OriginalExpirationDate value={value} setValue={setValue} />
+          {value ? <p>The selected date is {value.toLocaleString()}</p> : null}
```

Comment:
I'm not sure what the standard is, but at Twitter we used a ternary to avoid parsing null values. If value is populated, we will parse and show it. If it is not, we will return null, which will display nothing.